forge-gui/res/cardsfolder/upcoming/seething_skitter_priest.txt

```diff
@@ -0,0 +1,14 @@
+Name:Seething Skitter-Priest
+ManaCost:2 W B
+Types:Creature Phyrexian Cleric
+PT:3/3
+K:Toxic:1
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigEffect | TriggerDescription$ When CARDNAME enters the battlefield, creatures you control and creature cards in your hand perpetually gain "When this creature dies, create a 1/1 colorless Phyrexian Mite artifact creature token with toxic 1 and 'This creature can't block.'"
+SVar:TrigEffect:DB$ Effect | RememberObjects$ Valid Creature.YouCtrl,ValidHand Creature.YouOwn | StaticAbilities$ PerpetualTokenTrig | Name$ Seething Skitter-Priest's Perpetual Effect | Duration$ Permanent
+SVar:PerpetualTokenTrig:Mode$ Continuous | Affected$ Card.IsRemembered | AddTrigger$ DieToken | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ This creature perpetually gains "When this creature dies, create a 1/1 colorless Phyrexian Mite artifact creature token with toxic 1 and 'This creature can't block.'"
+SVar:DieToken:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When this creature dies, create a 1/1 colorless Phyrexian Mite artifact creature token with toxic 1 and 'This creature can't block.'"
```

Comment:
I was going to say remove the last " but Idk it's a bit funky. Maybe it makes sense to use only " on this line?

@Northmoc take a look at the oracle for this card, for lines 6, 8 and 14 it's fine but what do you think it should look like here on line 9? I know you like things to look nice/correct :D

Reply:
like this ?

Reply:
No, you changed the Oracle line. For grammer, that should look how it did before so that one needs reversing.

What I mean is, for the Oracle line and lines 6 & 8, they have a sentence and within that, some quotation marks and then within that another, hence the use of both " and '

But line 9 only uses the sentence within the first quotes so there is no need for both. I think do the following because if it's a lone sentence it should read like:
When this creature dies, create a 1/1 colorless Phyrexian Mite artifact creature token with toxic 1 and "This creature can't block."

Sorry if I'm not making any sense, DM if needed. (remember to revert Oracle back to):

When Seething Skitter-Priest enters the battlefield, creatures you control and creature cards in your hand perpetually gain "When this creature dies, create a 1/1 colorless Phyrexian Mite artifact creature token with toxic 1 and 'This creature can't block.'"

Reply:
Ah I understand now. Thanks !